021-changelog-upgrade.md

```diff
@@ -15,7 +15,7 @@ Changelog is written for each version released. The file name is `CHANGELOG.md`.
 
 ## 1.0.1 March 23, 2021
 
-- Bug #42: Short description of the change (author1, author2)
+- Bug #42: Short description of the change (@author1, @author2)
```

Comment:
```suggestion
- Bug #42: Short description of the change (`@author1`, `@author2`)
```

Reply:
Not need wrap nicknames as code: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams